src/main/java/com/odysseusinc/arachne/portal/service/analysis/heracles/HeraclesAnalysisKind.java

```diff
@@ -0,0 +1,97 @@
+package com.odysseusinc.arachne.portal.service.analysis.heracles;
+
+import static com.odysseusinc.arachne.portal.service.analysis.heracles.HeraclesConstants.HERACLES_FULL_ANALYSES_SET;
+import static com.odysseusinc.arachne.portal.service.analysis.heracles.HeraclesConstants.INCLUDE_DRUG_TYPE_UTILIZATION_DEFAULT;
+import static com.odysseusinc.arachne.portal.service.analysis.heracles.HeraclesConstants.INCLUDE_VISIT_TYPE_UTILIZATION_DEFAULT;
+
+public enum HeraclesAnalysisKind {
+
+    FULL(HERACLES_FULL_ANALYSES_SET,
```

Comment:
can you also add jdoc that there are  other types and meanwhile we added only one?

Reply:
done